go.mod

```diff
@@ -129,3 +129,5 @@ require (
 )
 
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20210831091943-07e756545ac1
+
+replace github.com/3scale/3scale-porta-go-client v0.11.0 => github.com/MStokluska/3scale-porta-go-client v0.0.0-20250124150520-23225e662421
```

Comment:
Will need to cut a 3scale-porta-go-client release and change this before merge.

Reply:
Stating the obvious here.

Reply:
yea, once you verify and happy about it we can proceed

Reply:
Hey @MStokluska should this have merged without the go mod update ?